src/mitol/hubspot_api/__init__.py

```diff
@@ -1,5 +1,5 @@
 """ mitol.hubspot_api """
 default_app_config = "mitol.hubspot_api.apps.HubspotApiApp"
 
-__version__ = "1.0.1"
-__distributionname__ = "mitol-django-hubspot_api"
+__version__ = "0.1"
```

Comment:
Is this right (`0.1`)?  Current published version is `1.0.0`

Reply:
Oh, meant to undo that.

Reply:
Fixed, also fixed the version in the changelog.